Crypto/css/style.css

```diff
@@ -0,0 +1,1029 @@
+    margin-bottom: 48px;
+}
+
+/* Section 4 */
+
+.section4-item-container {
+    display: flex;
+    justify-content: space-between;
+    max-width: 1240px;
+    margin-left: auto;
+    margin-right: auto;
+}
+
+.section4-left-item {
+    margin-top: 100px;
+    display: block;
+    position: relative;
+}
+
+.section4-left-item-elips img {
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%, -50%);
+    display: block;
+    position: absolute;
+    z-index: -1;
+    width: auto;
+    height: auto;
+}
+
+.section4-left-item-iphone img {
+    max-width: 100%;
+    height: auto;
+}
+
+.section4-right-item {
+    display: block;
+    margin-top: 215px;
+    max-width: 574px;
+}
+
+.section4-right-item-post {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+
+}
+
+.section4-right-item-post img {
+    width: 24px;
+    height: 24px;
+}
+
+.section4-right-item h1 {
+    font-family: var(--font-primary);
+    font-style: normal;
+    font-weight: 500;
+    font-size: 20px;
+    line-height: 28px;
+    letter-spacing: 0.16em;
+    text-transform: uppercase;
+    color: #C23893;
+}
+
+.section4-right-item h2 {
+    font-family: var(--font-primary);
+    font-style: normal;
+    font-weight: 500;
+    font-size: 60px;
+    line-height: 48px;
+    text-transform: capitalize;
+    color: #FFFFFF;
+}
+
+.section4-right-item-post h3 {
+    display: flex;
+    align-items: center;
+    gap: 8px;
+    font-family: var(--font-primary);
+    font-style: normal;
+    font-weight: 500;
+    font-size: 18px;
+    line-height: 28px;
+    text-transform: capitalize;
+    color: #FFFFFF;
+    margin-top: 32px;
+}
+
+.section4-right-item p {
+    font-family: var(--font-secondary);
+    font-style: normal;
+    font-weight: 400;
+    font-size: 18px;
+    line-height: 168.5%;
+    color: #767799;
+    margin-top: 11px;
+}
+
+/* Section 5 */
+
+.section5-wrapper {
+    display: flex;
+    flex-direction: row;
+    gap: 71px;
+    margin-top: 25px;
+}
+
+.section5-item-left {
+    display: block;
+    max-width: 603px;
+}
+
+.section5-item-left h1 {
+    font-family: var(--font-primary);
+    font-style: normal;
+    font-weight: 500;
+    font-size: 60px;
+    line-height: 104.5%;
+    color: #FFFFFF;
+}
+
+.section5-item-left p {
+    font-family: var(--font-secondary);
+    font-style: normal;
+    font-weight: 400;
+    font-size: 18px;
+    line-height: 168.5%;
+    color: #767799;
+    margin-top: 32px;
+}
+
+.section5-item-left a {
+    display: inline-block;
+    border: 1px solid #FFFFFF;
+    text-decoration: none;
+    padding: 20px 44.5px 20px 44.5px;
+    font-family: var(--font-secondary);
+    font-style: normal;
+    font-weight: 600;
+    font-size: 21px;
+    line-height: 25px;
+    text-transform: capitalize;
+    color: #FFFFFF;
+    margin-top: 52px;
+}
+
+.section5-item-right {
+    display: block;
+    position: relative;
+}
+
+.section5-item-right-ellipse img {
+    display: block;
```

Comment:
add width/height for all ellipse images, if you upload bigger images of the ellipse it will break your markup.